### PR TITLE
Enable Publishing to Ballerina Central on Release

### PR DESCRIPTION
**Pull request by MohamedSabthar**

### CODE REVIEW

File: udp-ballerina/build.gradle [hunk 1]
```diff
@@ -264,35 +234,49 @@ task ballerinaBuild {
...
-        }
-    }
-
-    repositories {
-        maven {
-            name = "GitHubPackages"
-            url = uri("https://maven.pkg.github.com/ballerina-platform/module-ballerina-udp")
-            credentials {
-                username = System.getenv("packageUser")
-                password = System.getenv("packagePAT")
```
Comment (ThisaruGuruge) [inline udp-ballerina/build.gradle:290]: Why these tasks removed?

File: udp-ballerina/build.gradle [hunk 2]
```diff
@@ -219,8 +203,8 @@ task ballerinaBuild {
             }
         }
         copy {
-            from file("$project.projectDir/target/balo")
-            into file("$artifactCacheParent/balo/${packageOrg}/${packageName}/${tomlVersion}")
+            from file("$project.projectDir/target/bala")
+            into file("$artifactCacheParent/bala/${packageOrg}/${packageName}/${tomlVersion}")
```
Comment (ThisaruGuruge) [inline udp-ballerina/build.gradle:207]: Why do we have to do the `balo` to `bala` change again?

File: udp-ballerina/build.gradle [hunk 3]
```diff
@@ -113,9 +113,9 @@ task copyStdlibs(type: Copy) {
 
 task updateTomlFile {
     doLast {
-        def stdlibDependentIoVersion = project.stdlibIoVersion.split("-")[0]
-        def stdlibDependentLogVersion = project.stdlibLogVersion.split("-")[0]
-        def nettyVersion = project.nettyVersion.split("-")[0]
+        def stdlibDependentIoVersion = project.stdlibIoVersion.replace("${snapshotVersion}", "")
+        def stdlibDependentLogVersion = project.stdlibLogVersion.replace("${snapshotVersion}", "")
+        def nettyVersion = project.nettyVersion.replace("${snapshotVersion}", "")
```
Comment (ThisaruGuruge) [inline udp-ballerina/build.gradle:118]: We don't need to replace `SNAPSHOT` in netty version. It should be a stable version always.

File: udp-ballerina/build.gradle (hunk 1 above)
Comment (MohamedSabthar) [reply udp-ballerina/build.gradle:290]: added again

File: udp-ballerina/build.gradle (hunk 3 above)
Comment (MohamedSabthar) [reply udp-ballerina/build.gradle:118]: removed `replace(..`

File: udp-ballerina/build.gradle (hunk 2 above)
Comment (MohamedSabthar) [reply udp-ballerina/build.gradle]: this is changed now

**Review by Maninda — APPROVED** [review]

**Pull request merged**